Dockerfile

```diff
@@ -40,4 +41,4 @@ COPY --from=builder --chown=appuser:appgroup /app/applicationinsights.dev.json /
 
 USER 2000
 
-ENTRYPOINT ["java", "-javaagent:/app/agent.jar", "-jar", "/app/app.jar"]
+ENTRYPOINT ["java", "-XX:+AlwaysActAsServerClassMachine", "-javaagent:/app/agent.jar", "-jar", "/app/app.jar"]
```

Comment:
intrigued why you've added this in?

Reply:
Probably don't needed it, I think it prevents auto detection what type of machine it is, improves start up performance may be, we just used it on other projects will check with the other devs why we included this.